locations/spiders/thecheesecakefactory.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from scrapy.spiders import SitemapSpider
+
+from locations.hours import OpeningHours
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class UrbanOutfitters(SitemapSpider, StructuredDataSpider):
```

Comment:
Hi Tony, One small comment would be to change the class name to TheCheesecakeFactorySpider. Cheers, Joe.
PS. The StructuredDataSpider is pretty cool I think good to see you're having some "fun" with it.

Reply:
Thanks for reviewing @mjoe999; it is now fixed.

Regarding the StructureDataSpider, it's excellent to reduce repeating ourselves so much.